src/Generator/ValueGenerator.php

```diff
@@ -103,6 +104,11 @@ public function __construct(
 
     /**
      * Init constant list by defined and magic constants
+     *
+     * @deprecated this method attempts to make some magic constants work with the value generator,
```

Comment:
Is the intention that these will be removed in `5.x`? Maybe mention here (as you have elsewhere) that it will be removed in the next major release?

Reply:
I will do a sweep for killing everything that is `@deprecated` after `4.8.0` is released :)

Reply:
Raised @ https://github.com/laminas/laminas-code/issues/155